src/main/java/de/stedo/shopService/ShopService.java

```diff
@@ -1,9 +1,58 @@
 package de.stedo.shopService;
 
+import java.util.List;
 import java.util.Map;
 
 public class ShopService {
 
+    private ProductRepo databaseProducts;
+    private OrderRepo databaseOrders;
 
+    public ShopService(ProductRepo databaseProducts, OrderRepo databaseOrders) {
+        this.databaseProducts = databaseProducts;
+        this.databaseOrders = databaseOrders;
+    }
+
+    public void add(int orderId, String name, List<Integer> artikelIds) {
+
+        artikelIds.forEach(Integer -> {
+            boolean testArtikelIds = databaseProducts.getProductList().containsKey(Integer);
+            if (!testArtikelIds) {
+                throw new RuntimeException("Diesen Artikel gibt es nicht!");
+            }
+        });
+
+        Order order = new Order(orderId, name, artikelIds);
+        databaseOrders.getOrderList().put(orderId, order);
+    }
+
+    public String getProduct(int id) {
+        return databaseProducts.getProduct(id).getProductName();
+    }
+
+    public String listProducts() {
```

Comment:
Die Idee war eigentlich, dass diese Methode ein `Product` Objekt zurückgibt und nicht nur den Namen. Dem Aufrufer fehlen ja sonst einige Informationen wie die Produkt IDs in der Order.
Fur den Fall, dass das `Product` nicht vorhanden ist, kannst du `null` zurückgegeben oder du guckst dir Mal die Klasse `Optional` an.